buildSrc/src/main/kotlin/Project.kt

```diff
@@ -2,7 +2,7 @@ object AndroidVersions {
   const val minSdkVersion = 21
   const val minAndroidAutoSdkVersion = 23
   const val targetSdkVersion = 30
-  const val compileSdkVersion = 30
+  const val compileSdkVersion = 31
```

Comment:
```
 ..1.7.0/res/values/values.xml:105:5-114:25: AAPT: error: resource android:attr/lStar not found.
```
Receiving the above error when using the android auto car library 1.1.0. In order to use the stable release version it seems we need to also update the compileSdkVersion

Reply:
thanks for flagging, this means that upstream will have to comply as well. Will work on that

Reply:
@tobrun any progress here? There wasn't much to change to update, only needed to update circleci

```
+ - image: mbgl/android-ndk-r22:latest
- - image: mbgl/android-ndk-r21e:latest
```

Reply:
@kmadsen we can use the android-ndk-r21e image, as @kiryldz has added build tool version 31 to that docker image: https://github.com/mapbox/mbgl-ci-images/pull/68

Reply:
Reverting to `mbgl/android-ndk-r21e` and let the ci/cd run

Reply:
It failed when using `r21e` 🤔 

https://app.circleci.com/pipelines/github/mapbox/mapbox-maps-android/3962/workflows/17f8a7cd-8a65-4f8e-bfee-e9cdff8049ef/jobs/30861

Reply:
`21e` is still broken

Reply:
It seems fail is due to the api-check that was recently changed 
```
+ ./scripts/semver-check.sh '' 10.3.0-beta.1 patch
Semver-check TAGGED_RELEASE_VERSION:, LAST_VERSION: 10.3.0-beta.1, API_COMPAT_LEVEL: patch
Release tagged with correct version.
+ rm -rf /tmp/tmp.j6R3RaaXo0
+ [[ -f ./scripts/../api_compat_report/major.txt ]]
+ echo 'Potential major level breaking change found:'
Potential major level breaking change found:
+ cat ./scripts/../api_compat_report/major.txt
style 
+ exit 1
```

Reply:
interesting this only fails when running on `21e` and succeeds on `22` 🤔

Reply:
oh wait, it's failing on both images now, i'll go back to `21e`

Reply:
Well now it's succeeding. Maybe this is just flaky? resolving this thread